### PR TITLE
IDE-1913 Error occurs when create maven project using IDE-3.0

### PR DESCRIPTION
**Pull request by aquilabj**

### CODE REVIEW

File: common/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/PropertiesUtil.java [hunk 1]
```diff
@@ -515,7 +516,7 @@ public void startElement( String uri, String localName, String qName, Attributes
         final ILiferayProject lrproject = LiferayCore.create( proj );
         final IFolder[] srcFolders = lrproject.getSourceFolders();
 
-        if( srcFolders.length < 1 )
+        if( CoreUtil.isNullOrEmpty( srcFolders ) || srcFolders.length < 1 )
```
Comment (gamerson) [inline common/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/PropertiesUtil.java]: The core util already checks for length<1

**Comment by gamerson** [comment]
Closing see comment.

File: common/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/PropertiesUtil.java (hunk 1 above)
Comment (aquilabj) [inline common/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/PropertiesUtil.java]: Sorry,  I will be careful later.

Comment (gamerson) [inline common/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/PropertiesUtil.java]: No problem

On Thu, Mar 5, 2015 at 1:21 PM, simon.jiang <notifications@github.com>
wrote:

> In
> common/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/PropertiesUtil.java
> <https://github.com/gamerson/liferay-ide/pull/926#discussion_r25840335>:
>
> > @@ -515,7 +516,7 @@ public void startElement( String uri, String localName, String qName, Attributes
> >          final ILiferayProject lrproject = LiferayCore.create( proj );
> >          final IFolder[] srcFolders = lrproject.getSourceFolders();
> >
> > -        if( srcFolders.length < 1 )
> > +        if( CoreUtil.isNullOrEmpty( srcFolders ) || srcFolders.length < 1 )
>
> Sorry, I will be careful later.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gamerson/liferay-ide/pull/926/files#r25840335>.
>



-- 
Greg Amerson
Liferay Developer Tools
Liferay, Inc. www.liferay.com

**Pull request closed without merging**